client/luigi-client.d.ts

```diff
@@ -197,11 +197,11 @@ export declare interface LinkManager {
   withParams: (nodeParams: NodeParams) => this;
 
   /**
-   * Opens a microfrontend as a modal
-   * @param {string} path path to be navigated to
-   * @param {Object} modalSettings settings to customize the modal title and size
-   * @param {string} modalSettings.title modal title
-   * @param {string} modalSettings.size size of the modal (l=large 80% default, m=medium 60%, s=small 40%)
+   * Opens a view in a modal. You can specify the modal's title and size. If you don't specify the title, it is the node label. If there is no node label, the title remains empty.  The default size of the modal is large (`l`), which means 80%. You can also use `m` (60%) and `s` (40%) to set the modal size. Optionally, use it in combination with any of the navigation functions.
+   * @param {string} path navigation path
```

Comment:
```suggestion
   * @param {string} path navigation path
```
```suggestion
   * @param {string} navigation path
```
I think it was one path too many :)

Reply:
I guess we cannot change it because the first 'path' is the parameter and 'navigation path' is the description of the parameter.

Reply:
True, missed this one. It's all fine :)